Firmware/mmu.cpp

```diff
@@ -181,6 +181,18 @@ bool check_for_ir_sensor()
 #endif //IR_SENSOR
 }
 
+static bool activate_stealth_mode()
```

Comment:
#define MMU_STEALTH_MODE has misleading name, it sounds like it is enabling stealth mode support, but in reality it is forcing it, I would propose to rename it to something like MMU_FORCE_STEALTH_MODE

There are three possible alternatives of how activate_stealth_mode() behave, but it is not clear from the source code at the first sight and in case both defines are enabled, first action is redundant.
I would propose something like:

#if defined (MMU_STEALTH_MODE)
	return true;
#elif defined (SILENT_MODE_STEALTH)
	return (eeprom_read_byte((uint8_t*)EEPROM_SILENT) == SILENT_MODE_STEALTH);
#else
	return false;
#endif